src/rgw/rgw_op.cc

```diff
@@ -3130,8 +3130,7 @@ int RGWPutObjProcessor_Multipart::do_complete(size_t accounted_size,
 
   store->obj_to_raw(s->bucket_info.placement_rule, meta_obj, &raw_meta_obj);
 
-  r = store->omap_set(raw_meta_obj, p, bl);
-
+  r = store->omap_set(raw_meta_obj, p, bl, true);
```

Comment:
this could use a clarifying comment, ex:
```c++
  const bool must_exist = true; // detect races with abort
  r = store->omap_set(raw_meta_obj, p, bl, must_exist);
```

Reply:
@cbodley 
great idea!